### DISCUSSION TITLE
autostart wg0 at boot

### DISCUSSION THREAD

**Discussion by frezeen** [WGDashboard/WGDashboard#380] [Q&A]
just a simple or stupid question sorry:

how can i set wg0 to autostart at boot? everytime status is off when reboot

**Comment by NoirPi** [comment]
easiest way to achieve that would be a systemd service 
https://www.ivpn.net/knowledgebase/linux/linux-autostart-wireguard-in-systemd/

I used these instructions and it worked flawless
